docs/install/upgrade-installation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Upgrading your installation"
+weight: 21
+type: "docs"
+---
+
+If you already have Knative installed, upgrade your installation by running the
+install command for the new version. We recommend upgrading by a single version
+number. For example, if you have v0.6.0 installed, you should upgrade to v0.7.0
```

Comment:
Instead of "single version number" I would say "single _minor_ version". Multiple patch versions (i.e. v0.6.0 to v0.6.4) should be fine.

v{major}.{minor}.{patch} is common terminology.